source/lambda/apiprocessor/lambda_function.py

```diff
@@ -85,6 +83,24 @@ def lambda_handler(event, context):
                     request["documentId"] = event['queryStringParameters']['documentId']
                 result = search(request)
 
+        # a specific document redaction items (terms, headers and footers)
+        elif(event['resource'] == '/redaction'):
```

Comment:
I believe there is a redact/ API that is defined in the infrastructure but not being used in the apiprocessor.
Could we just use that?